marlowe-chain-sync/gen/Language/Marlowe/Runtime/ChainSync/Gen.hs

```diff
@@ -163,7 +163,7 @@ instance Arbitrary PolicyId where
   arbitrary = PolicyId . unScriptHash <$> arbitrary
 
 instance Arbitrary TokenName where
-  arbitrary = TokenName <$> genBytes
+  arbitrary = TokenName . BS.take 32 <$> genBytes
```

Comment:
Someday I'd like to separate our `Arbitrary` instances into ones that are unconstrained versus ones that have ledger or other constraints.

Reply:
Ideally, our types should enforce ledger constraints where appropriate via smart constructors.